CHANGELOG.md

```diff
@@ -15,10 +15,10 @@ and this project adheres to
   `From<Int128> for i128` ([#2268])
 - cosmwasm-std: Implement `Uint128::from_{be,le}_bytes` and
   `Uint64::from_{be,le}_bytes`. ([#2269])
-- cosmwasm-std: Added new `Ibc2Msg` and `CosmosMsg::Ibc2` variant ([#2390],
-  [#2403])
+- cosmwasm-std: Added new `Ibc2Msg` and `CosmosMsg::Ibc2` variant ([#2340],
+  [#2390], [#2403])
```

Comment:
This one isn't related to the PR. The link for #2390 existed in the changelog, but wasn't used anywhere and I think this is the entry where it should probably be.